plugins/woocommerce/src/Database/Migrations/MigrationHelper.php

```diff
@@ -262,4 +263,29 @@ private static function migrate_country_states_for_orders( string $country_code,
 
 		// phpcs:enable WordPress.DB.PreparedSQL.NotPrepared, WordPress.DB.PreparedSQL.InterpolatedNotPrepared
 	}
+
+	/**
+	 * Migrate state codes in the tax rates table.
+	 *
+	 * @param string $country_code The country that has the states for which the migration is needed.
+	 * @param array  $old_to_new_states_mapping An associative array where keys are the old state codes and values are the new state codes.
+	 * @return void
+	 */
+	private static function migrate_country_states_for_tax_rates( string $country_code, array $old_to_new_states_mapping ): void {
+		global $wpdb;
+
+		// phpcs:disable WordPress.DB.PreparedSQL.NotPrepared
```

Comment:
I think if you put the `$wpdb->prepare()` directly into the `$wpdb->query()`, the phpcs disabling won't be necessary. 

It's probably true for some other ones above, but this is fairly simple to change.

Reply:
I'll change it but in general I prefer to first generate the query in a separate variable and then run it, as the resulting code is cleaner than a nested `query(prepare($sql))`, even if it's at the expense of having to disable the check.